src/Generator/EpubGenerator.php

```diff
@@ -50,7 +51,7 @@ public function create( Book $book ) {
 		$oldBookTitle = $book->title;
 		$css = $this->getCss( $book );
 		$this->i18n = Util::getI18n( $book->lang );
-		setlocale( LC_TIME, $book->lang . '_' . strtoupper( $book->lang ) . '.utf8' );
+		setlocale( LC_TIME, $book->getLocale() . '.utf8' );
```

Comment:
We could probably remove this if we do not have anything else relying on the current locale.

Reply:
Good point, done.